tools/Pandell.targets

```diff
@@ -0,0 +1,827 @@
+<!--
+
+Pandell common targets file
```

Comment:
Please add a note explaining this file comes from `node-repo-init` and what the current version at the time of copy was.